.gitignore

```diff
@@ -21,6 +21,22 @@ stt_engines/snowboy/snowboydetect.py
 
 stt_engines/snowboy/snowboydetect.pyc
 
+stt_engines/deepspeech/audio/
+
+stt_engines/deepspeech/bin/
+
+stt_engines/deepspeech/deepspeech-0.7.0-models.pbmm
+
+stt_engines/deepspeech/deepspeech-0.7.0-models.scorer
+
+stt_engines/deepspeech/deepspeech.log
+
+stt_engines/deepspeech/include/
+
+stt_engines/deepspeech/lib/
+
+stt_engines/deepspeech/share/
+
```

Comment:
That should be everything that's added automatically during the DeepSpeech installation or by running DeepSpeech